core/src/main/scala/cats/syntax/representable.scala

```diff
@@ -0,0 +1,18 @@
+package cats
+package syntax
+
+trait RepresentableSyntax {
+  implicit final def catsSyntaxTabulate[A, R](f: R => A): TabulateOps[A, R] =
+    new TabulateOps[A, R](f)
+
+  implicit final def catsSyntaxIndex[F[_], A, R](fa: F[A])(implicit R: Representable.Aux[F, R]): IndexOps[F, A, R] =
+    new IndexOps[F, A, R](fa)
+}
+
+final class IndexOps[F[_], A, R](val fa: F[A]) extends AnyVal {
```

Comment:
Should this have R on the class? Maybe it doesn’t matter but it seems like it should be on index.

Reply:
In my tests it didn't seem to make a difference, but I think you're right, it makes more sense to put it on index. I'll make that change now.